test/test_css_parser_offset_capture.rb

```diff
@@ -18,30 +18,31 @@ def test_capturing_offsets_for_local_file
 
     # check that we found the body rule where we expected
     assert_equal 0, rules[0].offset.first
-    assert_equal 43, rules[0].offset.last
+    assert_equal 41, rules[0].offset.last
     assert_equal file_name, rules[0].filename
 
     # and the p rule
     assert_equal 45, rules[1].offset.first
-    assert_equal 63, rules[1].offset.last
+    assert_equal 61, rules[1].offset.last
     assert_equal file_name, rules[1].filename
   end
 
   # http://github.com/premailer/css_parser/issues#issue/4
   def test_capturing_offsets_from_remote_file
     # TODO: test SSL locally
+    # TODO: cache request to make test not require internet (and so much faster)
```

Comment:
add webmock so that will just break and we return stubbed replies ?

Reply:
I think webmock here is the right choice too.